symplyphysics/laws/astronomy/speed_of_rocket_close_to_speed_of_light_depends_on_mass_and_impulse.py

```diff
@@ -0,0 +1,50 @@
+from sympy import (Eq, solve)
```

Comment:
Add `relativistic` subfolder and move it there. Remove `close_to_speed_of_light` from the name.